telethon/extensions/html.py

```diff
@@ -9,8 +9,8 @@
 from ..tl.types import (
     MessageEntityBold, MessageEntityItalic, MessageEntityCode,
     MessageEntityPre, MessageEntityEmail, MessageEntityUrl,
-    MessageEntityTextUrl
-)
+    MessageEntityTextUrl,
+    MessageEntityMentionName)
```

Comment:
Can you move this to the above line and leave the parenthesis on its own, as it was before?